### PR TITLE
Fix dark menus in canvas, not in top toolbar

### PR DESCRIPTION
**Pull request by jasmussen**
Fixes small oversight in https://github.com/WordPress/gutenberg/pull/22351#issuecomment-630830889

The block editor uses elevation sparingly. Popovers by default have a shadow and ligther border because they, and modal dialogs, are elevated. But popovers in the canvas are not, and scroll under the top toolbar with the block. For those cases, we have the "alternate" display which uses block toolbar material to indicate the popover is part of that.

This PR fixes the typo that made all of them block toolbars, and fixes a couple of other cases where the old menus were used:

Screenshot 2020-05-21 at 10 34 38

Screenshot 2020-05-21 at 10 39 29

Screenshot 2020-05-21 at 10 44 25

Screenshot 2020-05-21 at 10 46 17

Screenshot 2020-05-21 at 11 04 57

### CODE REVIEW

File: packages/block-editor/src/components/block-alignment-toolbar/index.js
Comment (mtias) [inline packages/block-editor/src/components/block-alignment-toolbar/index.js:48]: What's the point of having this as a constant?

Comment (jasmussen) [reply packages/block-editor/src/components/block-alignment-toolbar/index.js:48]: It's the only way I found to get this prop to take.

Comment (youknowriad) [reply packages/block-editor/src/components/block-alignment-toolbar/index.js:48]: I think the advantage is that the object is not regenerated on each render. On expensive pure components, it could have an impact but maybe not everywhere.

Comment (jasmussen) [reply packages/block-editor/src/components/block-alignment-toolbar/index.js:48]: Happy to do which ever you agree on. Just haven't been able to get it to work without this method.

**Comment by ZebulanStanphill** [comment]
>But popovers [...] scroll under the top toolbar with the block.

[This may cease to be the case](https://github.com/WordPress/gutenberg/pull/21056#pullrequestreview-415650603) (at least temporarily until a better solution is found) once #21056 is merged.

File: packages/block-editor/src/components/block-alignment-toolbar/index.js
Comment (jasmussen) [reply packages/block-editor/src/components/block-alignment-toolbar/index.js:48]: It would be nice to unblock this PR, so we don't ship the current change in master if we can avoid it. CC: @youknowriad

Comment (youknowriad) [reply packages/block-editor/src/components/block-alignment-toolbar/index.js:48]: @jasmussen this shouldn't be a blocker IMO. You can inline these if needed. Performance is probably not a concern  here.

**Review by youknowriad — APPROVED** [review]

**Comment by jasmussen** [comment]
Thank you Riad. I'll merge this and address any follow-up that needs to be addressed.

**Pull request merged**
